README.md

```diff
@@ -59,33 +59,52 @@ To learn more about the platform, check out our [documentation](https://github.c
   ```sh
   git clone https://github.com/ostis-ai/ostis-web-platform --recursive
   cd ostis-web-platform
-  ./scripts/install_submodules.sh # download all submodules without compilation.
+  # download all submodules
+  ./scripts/install_submodules.sh
+  # build sc-machine, scp-machine and sc-component-manager
   docker compose build
   ```
 
    </details>
 
-- Natively (using sc-component-manager)
-
-  ```sh
-  git clone https://github.com/ostis-ai/ostis-web-platform
-  cd ostis-web-platform
-  ./scripts/install_minimal_platform.sh
-  ./scripts/run_sc_machine.sh
-  # and write write the following commands in the terminal:
-  # components init
-  # components install sc_web
-  # components install knowledge_base_ims
-  ```
-
 - Natively
 
-  Note: Currently, only Linux (Ubuntu-20.04, Ubuntu-22.04) and macOS are supported by this installation method. If you're going to use it, it might take a while to download dependencies and compile the components. Use it only if you know what you're doing!
+  Note: Currently, only Linux (Ubuntu-22.04, Ubuntu-24.04) and macOS are supported by this installation method.
 
   ```sh
   git clone https://github.com/ostis-ai/ostis-web-platform --recursive
   cd ostis-web-platform
-  ./scripts/install_platform.sh
+  ./scripts/install_submodules.sh
+
+  # to build sc-machine, see https://ostis-ai.github.io/sc-machine/build/quick_start/#start-develop-sc-machine-with-conan
+  cd sc-machine
+  # make sure, that you have `conan`, updated `cmake` and `ninja`
+  cmake --preset debug-conan
+  cmake --build --preset debug
+  conan export-pkg .
```

Comment:
![image](https://github.com/user-attachments/assets/d1f10a91-a3b7-4eef-a162-271e59ecc165)
got this error